### PR TITLE
Fix support for MOI.TimeLimitSec

### PR DESCRIPTION
**Pull request by odow**
x-ref https://github.com/jump-dev/MathOptInterface.jl/pull/2256

### CODE REVIEW

**Comment by matbesancon** [comment]
the issue here seems to go beyond the time limit?

**Comment by odow** [comment]
Seems unrelated? Dunno

**Comment by odow** [comment]
Same failure is here: https://github.com/mariohsouto/ProxSDP.jl/pull/100

File: test/moitest.jl [hunk 1]
```diff
@@ -69,6 +69,8 @@ function test_runtests()
             # time limit hit
             "test_linear_INFEASIBLE",
             "test_solve_DualStatus_INFEASIBILITY_CERTIFICATE_VariableIndex_LessThan_max",
+            # TODO(odow): investigate
+            "test_HermitianPSDCone_basic",
```
Comment (odow) [inline test/moitest.jl:73]:
```suggestion
            "test_HermitianPSDCone_",
```

File: test/moitest.jl [hunk 2]
```diff
@@ -69,6 +69,8 @@ function test_runtests()
             # time limit hit
             "test_linear_INFEASIBLE",
             "test_solve_DualStatus_INFEASIBILITY_CERTIFICATE_VariableIndex_LessThan_max",
+            # TODO(odow): investigate
+            "test_HermitianPSDCone_",
```
Comment (odow) [inline test/moitest.jl:73]:
```suggestion
            "test_HermitianPSDCone_",
            "test_NormNuclearCone_VectorAffineFunction_with_transform",
```

**Comment by odow** [comment]
Bump @joaquimg

**Pull request merged**